### PR TITLE
Rerender

### PR DESCRIPTION
**Pull request by tschoonj**

### CODE REVIEW

**Comment by tschoonj** [comment]
@CJ-Wright if you're ok with this PR, feel free to merge it in.

Thanks!

**Comment by CJ-Wright** [comment]
ping @mariusvniekerk @scopatz

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1 
+  number: 1
```
Comment (CJ-Wright) [inline recipe/meta.yaml]: This most likely needs to be bumped

**Comment by tschoonj** [comment]
Build number bumped

**Comment by tschoonj** [comment]
Thanks!

**Pull request merged**